### DISCUSSION TITLE
Release of libp2p sub crates

### DISCUSSION THREAD

**Discussion by jxs** [libp2p/rust-libp2p#5351] [General]
Hi everyone, 
the following sub crates have released minor versions:
- libp2p-identify: `0.44.2`
- libp2p-ping: `0.44.1`
- libp2p-relay:  `0.17.2`
- libp2p-request-response: `0.26.2`
- libp2p-stream: `v0.1.0-alpha-1`
- libp2p-swarm : `0.44.2`
- libp2p-swarm-derive : `0.34.2`
- libp2p-upnp: `0.2.2`
- libp2p-websocket-websys: `0.3.2`

**Comment by dhuseby** [comment]
Thank you @jxs !!

**Comment by dariusc93** [comment]
Thanks for the release! I noticed that libp2p-stream `v0.1.0-alpha-1` havent been published. Was that intentional?

**Reply by jxs** [reply]
ah, thanks for the reminder Darius, I don't have publishing rights over `libp2p-stream` @thomaseizinger can you publish a release? Thanks!

**Reply by thomaseizinger** [reply]
I'll add the team as an owner.

**Reply by thomaseizinger** [reply]
Done.

**Reply by jxs** [reply]
Thanks Thomas! Published `libp2p-stream` `v0.1.0-alpha.1`

**Comment by DougAnderson444** [comment]
@jxs What is the plan for the release of libp2p so it can use these sub-releases?

**Reply by jxs** [reply]
> @jxs What is the plan for the release of libp2p so it can use these sub-releases?

Hi Doug, thanks for asking and sorry that it's not more explicit, we are waiting for https://github.com/libp2p/rust-libp2p/pull/5349 after that I'll release `libp2p` `0.54` and in the future will use a tracking issue and labels for this situation.

**Reply by jxs** [reply]
Turns out there were a couple more breaking PR's pending, I have updated the [milestone](https://github.com/libp2p/rust-libp2p/milestone/10). Meanwhile you don't need a `libp2p` minor upgrade (0.54) to use the patch upgraded subcrates you can just do `cargo update` and it will upgrade them as they are semver compatible.

**Reply by DougAnderson444** [reply]
Good point! That works too :)
